### PR TITLE
Added missing test case for invalid range usage

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Added missing test case for the invalid default_ranges_stats usage

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
@gargn Can you pls review the same.

**Comment by amitsrivastava78** [comment]
@gargn can you pls review the PR and let me know your comments

File: tensorflow/lite/python/lite_test.py [hunk 1]
```diff
@@ -481,6 +481,39 @@ def testDefaultRangesStats(self):
     self.assertTrue(([1, 16, 16, 3] == output_details[0]['shape']).all())
     self.assertTrue(output_details[0]['quantization'][0] > 0)  # scale
 
+  def testInvalidStatsUsage(self):
```
Comment (gargn) [inline tensorflow/lite/python/lite_test.py]: nit: Name the test `testDefaultRangesStatsInvalid` to follow the pattern of test names.

**Review by gargn — CHANGES_REQUESTED** [review]
I am unclear what this test is actually testing. It seems to be the same as `testDefaultRangesStats`.

File: tensorflow/lite/python/lite_test.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/python/lite_test.py]: Updated as per the review comments

**Comment by amitsrivastava78** [comment]
@gargn thanks for your review, this TC just tell that if you do not inference_type as QUANTIZED_UINT8
then the convertion does not happen to the qunatization format.I have made the changes requested by you, kindly approve

**Comment by amitsrivastava78** [comment]
@gargn & @rthadur , I have updated the code as per your comments, kindly check, if everything is ok, kindly approve the PR.

**Comment by amitsrivastava78** [comment]
@rthadur i have resolved the merge conflicts, can you please update the label to review requested.

**Comment by gargn** [comment]
I don't think this test case is necessary so I'm closing this PR.

**Pull request closed without merging**